client/models/article.coffee

```diff
@@ -13,17 +20,18 @@ module.exports = class Article extends Backbone.Model
 
   urlRoot: "#{sd.API_URL}/articles"
 
-  initialize: ->
-    @sections = new Sections @get 'sections'
-    @featuredPrimaryArtists = new Artists
-    @featuredArtists = new Artists
-    @mentionedArtists = new Artists
-    @featuredArtworks = new Artworks
-    @mentionedArtworks = new Artworks
-    @leadParagraph = new Backbone.Model text: @get('lead_paragraph')
-    @heroSection = new Section @get 'hero_section'
-    @on 'change:hero_section', => @heroSection.set @get 'hero_section'
-    @heroSection.destroy = @heroSection.clear
+  initialize: (attributes, options = {}) ->
+    unless options.simple
+      @sections = new Sections @get 'sections'
+      @featuredPrimaryArtists = new Artists
+      @featuredArtists = new Artists
+      @mentionedArtists = new Artists
+      @featuredArtworks = new Artworks
+      @mentionedArtworks = new Artworks
+      @leadParagraph = new Backbone.Model text: @get('lead_paragraph')
+      @heroSection = new Backbone.Model @get 'hero_section'
+      @on 'change:hero_section', => @heroSection.set @get 'hero_section'
+      @heroSection.destroy = @heroSection.clear
 
```

Comment:
In short, all of them are unnecessary except those which you pass/set manually. By passing a `simple` flag, we don't make any associations on initialize. Happy to go through this in person if you have more questions!